Source/rclUE/Public/rclcUtilities.h

```diff
@@ -161,6 +161,79 @@ static const TMap<TEnumAsByte<UROS2QoS>, rmw_qos_profile_t> QoS_LUT = {
     {UROS2QoS::System, rmw_qos_profile_system_default},
     {UROS2QoS::UnknownQoS, rmw_qos_profile_unknown}};
 
+UCLASS()
+class URRTimerManager : public UObject
+{
+    GENERATED_BODY()
+protected:
+    bool bEnabled = true;
+
+    //! Timer handler for periodic publisher
+    UPROPERTY(VisibleAnywhere, BlueprintReadOnly)
+    FTimerHandle TimerHandle;
+
+    FTimerDelegate Delegate;
+
+    FTimerDelegate TimerDelegate;
+
+    UWorld* World;
```

Comment:
This could be already fetched using `GetWorld()`?

Reply:
it maybe because TimerManager is garbage collected